src/formats/yarle/utils/filename-utils.ts

```diff
@@ -37,7 +37,7 @@ export const getResourceFileProperties = (workDir: string, resource: any): Resou
 
 	if (resource['resource-attributes'] && resource['resource-attributes']['file-name']) {
 		const fileNamePrefix = resource['resource-attributes']['file-name'].substr(0, 50);
-		fileName = fileNamePrefix.split('.')[0];
+		fileName = fileNamePrefix.includes('.') ? fileNamePrefix.split('.').slice(0, -1).join('.') : fileNamePrefix;
```

Comment:
I think we could make this a little more efficient and avoid searching through the string multiple times:

```
const lastIdx = fileNamePrefix.lastIndexOf('.');
fileName = lastIdx !== -1
    ? fileNamePrefix.slice(0, lastIdx)
    : fileNamePrefix;
```